eva-accession-release/src/main/resources/create_release_properties.sh

```diff
@@ -94,12 +96,12 @@ logging.level.uk.ac.ebi.eva.accession.release=INFO
+bsub -J \${jobname}_txts -w \${jobname} -o txts.log -e txts.err -M 8192 -R \"rusage[mem=8192]\" \"sort -V ${OUTPUT_FOLDER}${assemblyAccession}_deprecated_ids.unsorted.txt | uniq | gzip > ${OUTPUT_FOLDER}${assemblyAccession}_deprecated_ids.txt.gz; sort -V ${OUTPUT_FOLDER}${assemblyAccession}_merged_deprecated_ids.unsorted.txt | uniq | gzip > ${OUTPUT_FOLDER}${assemblyAccession}_merged_deprecated_ids.txt.gz\"
+bsub -J \${jobname}_current_vcf -w \${jobname} -o ${OUTPUT_FOLDER}current_vcf.log -e ${OUTPUT_FOLDER}current_vcf.err \"${SORT_SCRIPT_PATH} ${OUTPUT_FOLDER}${assemblyAccession}_current_ids_unsorted.vcf ${OUTPUT_FOLDER}${assemblyAccession}_current_ids.vcf; ${VCF_VALIDATOR_PATH} -i ${OUTPUT_FOLDER}${assemblyAccession}_current_ids.vcf -o ${OUTPUT_FOLDER}; ${VCF_ASM_CHECKER_PATH} -i ${OUTPUT_FOLDER}${assemblyAccession}_current_ids.vcf -f ${fasta} -a ${assembly_report} -o ${OUTPUT_FOLDER} -r text,summary ; ${BGZIP_PATH} ${OUTPUT_FOLDER}${assemblyAccession}_current_ids.vcf\"
+bsub -J \${jobname}_merged_vcf -w \${jobname} -o ${OUTPUT_FOLDER}merged_vcf.log -e ${OUTPUT_FOLDER}merged_vcf.err \"${SORT_SCRIPT_PATH} ${OUTPUT_FOLDER}${assemblyAccession}_merged_ids_unsorted.vcf ${OUTPUT_FOLDER}${assemblyAccession}_merged_ids.vcf; ${VCF_VALIDATOR_PATH} -i ${OUTPUT_FOLDER}${assemblyAccession}_merged_ids.vcf -o ${OUTPUT_FOLDER}; ${VCF_ASM_CHECKER_PATH} -i ${OUTPUT_FOLDER}${assemblyAccession}_merged_ids.vcf -f ${fasta} -a ${assembly_report} -o ${OUTPUT_FOLDER} -r text,summary ; ${BGZIP_PATH} ${OUTPUT_FOLDER}${assemblyAccession}_merged_ids.vcf\"
+bsub -J \${jobname}_tabix -w \"(\${jobname}_current_vcf && \${jobname}_merged_vcf)\" -o tabix.log -e tabix.err \"${TABIX_PATH} ${OUTPUT_FOLDER}${assemblyAccession}_current_ids.vcf.gz; ${TABIX_PATH}  ${OUTPUT_FOLDER}${assemblyAccession}_merged_ids.vcf.gz\"
+bsub -J \${jobname}_count -w \"(\${jobname}_txts && \${jobname}_current_vcf && \${jobname}_merged_vcf)\" -o count_ids.log -e count_ids.err \"echo '# Unique RS ID counts' > ${OUTPUT_FOLDER}README_rs_ids_counts.txt; ${COUNT_IDS_PATH} ${OUTPUT_FOLDER}${assemblyAccession}_current_ids.vcf.gz >> ${OUTPUT_FOLDER}README_rs_ids_counts.txt; ${COUNT_IDS_PATH} ${OUTPUT_FOLDER}${assemblyAccession}_merged_ids.vcf.gz >> ${OUTPUT_FOLDER}README_rs_ids_counts.txt; zcat ${OUTPUT_FOLDER}${assemblyAccession}_deprecated_ids.txt.gz | wc -l | sed 's/^/'${assemblyAccession}_deprecated_ids.txt.gz'\t/' >> ${OUTPUT_FOLDER}README_rs_ids_counts.txt; zcat ${OUTPUT_FOLDER}${assemblyAccession}_merged_deprecated_ids.txt.gz | cut -f 1 | uniq | wc -l | sed 's/^/'${assemblyAccession}_merged_deprecated_ids.txt.gz'\t/' >> ${OUTPUT_FOLDER}README_rs_ids_counts.txt\"
```

Comment:
I know you didn't change most of this line, but why are we not sorting the deprecated_ids and merged_deprecated_ids files? I cannot see any sort in the release pipeline.

Reply:
Those `.txt` files are sorted [here](https://github.com/EBIvariation/eva-accession/pull/168/files#diff-a8b61f207cb67bd5eb10ad0a5b586b2cR100)